src/tools/kdb/cmdline.cpp

```diff
@@ -517,7 +517,11 @@ kdb::Key Cmdline::createKey (int pos) const
 
 	if (!root.isValid ())
 	{
-		throw invalid_argument (name + " is not a valid keyname");
+		throw invalid_argument (name + " is not a valid keyname" +
+								"\n\n"
+								"Please make sure that the key's namespace is valid and not missing (see 'man elektra-namespaces').\n"
```

Comment:
true. i adjusted the wording a bit, the problem is basically when you try to use a non-cascaded key, as i initally kept wondering why creating a key test/key did not work (> as there is most likely no test namespace globally). user/test would have worked instead, as the user namespaces does exist. 

is it better to understand now? any special situations i'm not yet aware about?